mts/platform/deviceinfo/deviceinfoprovider.cpp

```diff
@@ -44,27 +45,21 @@ using namespace meegomtp1dot0;
 DeviceInfoProvider::DeviceInfoProvider()
     : m_batteryStatus(new BatteryStatus(this))
 {
-    QDeviceInfo di;
-
+    DeviceInfo deviceInfo;
     SsuDeviceInfo ssuDeviceInfo;
 
     m_serialNo = ssuDeviceInfo.deviceUid();
-
-    m_deviceVersion = QString("%1 HW: %2").arg(di.version(QDeviceInfo::Os))
-                      .arg(di.version(QDeviceInfo::Firmware));
-
-    m_manufacturer = di.manufacturer().isEmpty() ? m_manufacturer : di.manufacturer();
-    m_model = di.model().isEmpty() ? m_model : di.model();
+    m_deviceVersion = QString("%1").arg(deviceInfo.osVersion());
```

Comment:
Unnecessary string formatting now, could be just the deviceInfo.osVersion()?

Reply:
Mmm, yes. Done.